sample/benchmark/src/main/java/com/datadog/benchmark/sample/di/activity/DatadogActivityModule.kt

```diff
@@ -59,5 +94,29 @@ internal interface DatadogActivityModule {
     }
 }
 
+@SuppressLint("LogNotTimber")
+private fun createDatadogConfiguration(): Configuration {
+    val configBuilder = Configuration.Builder(
+        clientToken = BuildConfig.BENCHMARK_CLIENT_TOKEN,
+        env = BuildConfig.BUILD_TYPE
+    )
+        .setBatchSize(BatchSize.SMALL)
+        .setUploadFrequency(UploadFrequency.FREQUENT)
+
+    configBuilder.setBackpressureStrategy(
+        BackPressureStrategy(
+            CAPACITY_BACK_PRESSURE_STRATEGY,
+            { Log.w("BackPressure", "THRESHOLD REACHED!") },
+            { Log.e("BackPressure", "ITEM DROPPED $it!") },
```

Comment:
let's avoid screaming in the logs :)

```suggestion
            { Log.w("BackPressure", "Threshold reached") },
            { Log.e("BackPressure", "Item dropped: $it") },
```